### ISSUE TITLE
ethereum wallet not working in windows 10

### ISSUE THREAD

**Issue by liviumey** [ethereum/mist#3885]
* [ ] I've asked for help in the [Mist Gitter](http://gitter.im/ethereum/mist) before filing this issue.

```
Version: `0.10.0`
OS & Version: Windows 10
Node version: `geth 1.8.7`
Number of blocks synchronized: 97%
```

Doesnt sync. Chain structure never catches up.
![image](https://user-images.githubusercontent.com/39199494/39955704-82c5ca30-55dc-11e8-9ab1-f6c83984eac2.png)

If i start without geth it doesnt finds peers.
![image](https://user-images.githubusercontent.com/39199494/39955726-def6947e-55dc-11e8-9c0b-787afe2a1e9e.png)

**Comment by fiddybux** [comment]
I have found the same issue with Light Client, and sometimes, not always, it fixes itself after many hours. Other times I can fix it by switching to Solo Network mode and back to Main Network (Light Client) mode. It's all pretty random and positive outcomes are not repeatable, but you might want to try these things.

**Comment by arumor** [comment]
Yeah I opened my wallet today and finally they showed up again. 3 days of missing in action, Super frustrating. I'm not sure how I can support a coin who's network seems to have some major network issues and there is no support to be found anywhere. I even tried checking etherscan and nothing was showing up. It was like my coins never existed. Horrendous.
Jason
-------- Original message --------From: Fiddy Bux <notifications@github.com> Date: 2018-05-20  2:28 PM  (GMT-07:00) To: ethereum/mist <mist@noreply.github.com> Cc: Subscribed <subscribed@noreply.github.com> Subject: Re: [ethereum/mist] ethereum wallet not working in windows 10 (#3885) 
I have found the same issue with Light Client, and sometimes, not always, it fixes itself after many hours. Other times I can fix it by switching to Solo Network mode and back to Main Network (Light Client) mode. It's all pretty random and positive outcomes are not repeatable, but you might want to try these things.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or mute the thread.
{"@context":"http://schema.org","@type":"EmailMessage","potentialAction":{"@type":"ViewAction","target":"https://github.com/ethereum/mist/issues/3885#issuecomment-390509773","url":"https://github.com/ethereum/mist/issues/3885#issuecomment-390509773","name":"View Issue"},"description":"View this Issue on GitHub","publisher":{"@type":"Organization","name":"GitHub","url":"https://github.com"}}
{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/ethereum/mist","title":"ethereum/mist","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/ethereum/mist"}},"updates":{"snippets":[{"icon":"PERSON","message":"@fiddybux in #3885: I have found the same issue with Light Client, and sometimes, not always, it fixes itself after many hours. Other times I can fix it by switching to Solo Network mode and back to Main Network (Light Client) mode. It's all pretty random and positive outcomes are not repeatable, but you might want to try these things."}],"action":{"name":"View Issue","url":"https://github.com/ethereum/mist/issues/3885#issuecomment-390509773"}}}
{
"@type": "MessageCard",
"@context": "http://schema.org/extensions",
"hideOriginalBody": "false",
"originator": "37567f93-e2a7-4e2a-ad37-a9160fc62647",
"title": "Re: [ethereum/mist] ethereum wallet not working in windows 10 (#3885)",
"sections": [
{
"text": "",
"activityTitle": "**Fiddy Bux**",
"activityImage": "https://avatars3.githubusercontent.com/u/25105809?s=160\u0026v=4",
"activitySubtitle": "@fiddybux",
"facts": [

]
}
],
"potentialAction": [
{
"name": "Add a comment",
"@type": "ActionCard",
"inputs": [
{
"isMultiLine": true,
"@type": "TextInput",
"id": "IssueComment",
"isRequired": false
}
],
"actions": [
{
"name": "Comment",
"@type": "HttpPOST",
"target": "https://api.github.com",
"body": "{\n\"commandName\": \"IssueComment\",\n\"repositoryFullName\": \"ethereum/mist\",\n\"issueId\": 3885,\n\"IssueComment\": \"{{IssueComment.value}}\"\n}"
}
]
},
{
"name": "Close issue",
"@type": "HttpPOST",
"target": "https://api.github.com",
"body": "{\n\"commandName\": \"IssueClose\",\n\"repositoryFullName\": \"ethereum/mist\",\n\"issueId\": 3885\n}"
},
{
"targets": [
{
"os": "default",
"uri": "https://github.com/ethereum/mist/issues/3885#issuecomment-390509773"
}
],
"@type": "OpenUri",
"name": "View on GitHub"
},
{
"name": "Unsubscribe",
"@type": "HttpPOST",
"target": "https://api.github.com",
"body": "{\n\"commandName\": \"MuteNotification\",\n\"threadId\": 334584486\n}"
}
],
"themeColor": "26292E"
}

**Comment by fiddybux** [comment]
That's not really helpful on a mobile device where I am replying from now.

There is either too much content at #2466, or posts are not numbered on
small screens.

Can you paraphrase?

I've been able to get light client syncing by manually starting geth.exe
--syncmode "light", before the wallet GUI.

Still takes ages to find peers though.
